qt-downloader

```diff
@@ -117,10 +117,12 @@ def get_info(url, version, toolchain):
   return None
 
 
-def download_and_extract(archives_url, archives):
+def download_and_extract(archives_url, archives, output, req_modules):
```

Comment:
The `output` argument should be a part of the _Allow to Choose Output Directory_ commit.

Reply:
How do i add a part of line into a commit?

Reply:
I suggest amending the aforementioned commit to add this argument. And verify if the feature declared by the commit works. It does not as per now.

Reply:
Ok will do in some time. Thanks

Reply:
If you don't mind, I can do this housecleaning for all the cases except the _Allow to download Specific Toolchains_ commit since I have not addressed it yet.

Reply:
Yes feel free to modify my changes